Core/fft_mt_r2iq.cpp

```diff
@@ -19,19 +19,35 @@ The name r2iq as Real 2 I+Q stream
 
 #include "fir.h"
 
+#include <assert.h>
 #include <string.h>
 #include <algorithm>
-#include <assert.h>
+#include <utility>
+
+// assure, that ADC is not oversteered?
+#define PRINT_INPUT_RANGE  0
```

Comment:
i am not sure if this is useful. Oscar has a PR #140 which has hardware indicator for this. From what I tested, it is hard to overload ADC.

Reply:
I would not perfer adding the test code in the main logic. As we have libsddc now, this code will be easy to write a small utility to test it out.

Reply:
* i needed it for local testing .. to determine if ADC already oversteered from signal generator .. for now, the code is deactivated with `#define PRINT_INPUT_RANGE 0` anyway
* hardware indicator (for RX888) is not useful .. cause it isn't visible from outside
* later on, i would like to 'somehow' pass the result to HDSDR .. see https://github.com/ik1xpv/ExtIO_sddc/issues/140

Reply:
okay. looks good to me. Go ahead merge this.